fog/sql_recovery_db/cleanup/src/main.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Runs a cleanup utility for the Fog Sql Recovery DB.
+
+use crate::{config::SqlRecoveryDbCleanupConfig, db_cleaner::DbCleaner};
+use chrono::Duration;
+use clap::Parser;
+use mc_common::logger::create_app_logger;
+use mc_fog_sql_recovery_db::SqlRecoveryDb;
+use std::env;
+
+mod config;
+mod db_cleaner;
+
+static EXPIRATION_DAYS: i64 = 2;
```

Comment:
:thought_balloon: I think this could be a future PR if desired, should the expiration time and database_url be args as well.

Reply:
Sounds good.